tests/integration/models/reddit/test_live.py

```diff
@@ -0,0 +1,21 @@
+"""Test praw.models.LiveThread"""
+from praw.models import LiveThread
+import mock
+
+from ... import IntegrationTest
+
+
+class TestLiveThread(IntegrationTest):
+    @mock.patch('time.sleep', return_value=None)
+    def test_init(self, _):
+        thread_id = 'ukaeu1ik4sw5'
+        data = {'id': thread_id}
+
+        with self.recorder.use_cassette('TestLiveThread_test_init'):
+            thread = LiveThread(self.reddit, thread_id)
```

Comment:
If this is lazy we should probably put this line and the next outside the betamax block.